tools/core-build-tasks/src/tasks/helpers/getOrThrowFromProcess.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Mojang AB.  All rights reserved.
+
+/**
+ * Checks process.env for the desired key and returns the value if it exists, otherwise throws an error. Generic parameter
+ * is used to ensure the correct type is returned
+ *
+ * @param key - The key to query from process env
+ * @param message - Message to include in thrown error on failure
+ * @returns Value from process env or throws an error
+ */
+export function getOrThrowFromProcess<T = string>(key: string, messageOverride?: string): T {
+    const value = process.env[key];
+    if (value === undefined) {
```

Comment:
nit:
```suggestion
    if (!value) {
```

captures a few more cases, like if `value` is `null` for some reason

Reply:
Fixed, tx